apps/api/src/routes/v1_keys_createKey.happy.test.ts

```diff
@@ -467,4 +467,37 @@ describe("with externalId", () => {
       expect(key!.identity!.id).toEqual(identity.id);
     });
   });
+  describe("Should default last day of month if none provided", () => {
```

Comment:
No this was me being dumb and tired. It will default to first day. I was confusing with needing to check if day is past last day on the actual createKey route.